bitnami/etcd/templates/svc.yaml

```diff
@@ -15,32 +23,23 @@ spec:
   loadBalancerSourceRanges: {{- toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
   {{- end }}
   {{- if .Values.service.externalIPs }}
-  externalIPs:
-    {{ toYaml .Values.service.externalIPs | indent 4 }}
+  externalIPs: {{- toYaml .Values.service.externalIPs | nindent 4 }}
   {{- end }}
   ports:
-    {{- if .Values.auth.client.clientPortNameOverride }}
-    - name: {{ .Values.service.clientPortNameOverride }}
-    {{- else }}
-    - name: "client"
-    {{- end }}
+    - name: {{ ternary .Values.auth.client.clientPortNameOverride "client" (not (empty .Values.auth.client.clientPortNameOverride)) | quote }}
```

Comment:
Looking at this... doesn't the `default` function work better for this? I think that ternary makes sense if the boolean expression is different from checking that the same variable you are using in ternary exists or not

Reply:
Totally agree

Reply:
Please double check the rest of the `ternary` calls in the YAML